README.md

```diff
@@ -1,7 +1,6 @@
 # CircBuf
-
-[![Build Status](https://travis-ci.org/jeromefroe/circbuf-rs.svg?branch=master)](https://travis-ci.org/jeromefroe/circbuf-rs)
-[![codecov](https://codecov.io/gh/jeromefroe/circbuf-rs/branch/master/graph/badge.svg)](https://codecov.io/gh/jeromefroe/circbuf-rs)
+[![build status](https://github.com/prx0/circbuf-rs/actions/workflows/rust.yml/badge.svg)](https://github.com/prx0/circbuf-rs/actions/workflows/rust.yml)
```

Comment:
I think these URL's currently point to your fork and so we just need to replace `prx0` with `jeromefroe`.

Reply:
You're right! Thank you I changed these links.